bookList-no-cell/components/app-navbar/app-navbar.js

```diff
@@ -0,0 +1,29 @@
+import { LitElement, html } from "lit";
+
+import navbar from "./app-navbar.styles";
+
+export class appNavbar extends LitElement {
+  static properties = {
+  };
+
+  constructor() {
+    super();
+  }
```

Comment:
if you don't need properties, it's not necessary to define them, same with the constructor